herbert_core/applications/multifit/@mfclass/private/mask_points_for_fit_xye.m

```diff
@@ -25,7 +25,7 @@
 %           If give variances, this has the same effect (except that the function cannot check
 %           negative error bars; it does check negative variances of course)
 %
-%   mask    [Optional] array of ones and zeros, with the same number of elements as the data
+%   keep    [Optional] array of ones and zeros, with the same number of elements as the data
```

Comment:
again, why name it keep? what does it do?

Reply:
mask presumed removal of marked elements, and here we keep elements which are marked as true. 
This is why renaming occurs. It extreamly difficult to understand code, when you expect one action and it does opposite.

Reply:
at least looking at inconsistency between the method name (`mask`) and the argument name `keep` you think what is actually happening.

Reply:
Personally, I disagree, I find the concept of a positive or a negative mask equally valid. However, I think the name `keep` and certainly `keep_out` are worse names than mask. If anything `to_keep` for both would be better, however, I would argue for leaving as-is.